conda_build/convert.py

```diff
@@ -32,7 +32,7 @@
 @echo off
 set PYFILE=%~f0
 set PYFILE=%PYFILE:~0,-4%-script.py
-"%~f0\\..\\..\\python.exe" "%PYFILE%" %*
+"%~dp0\..\python.exe" "%PYFILE%" %*
```

Comment:
I'm not sure that the entry point scripts ever worked.  This expression was just wrong.